core/arch/arm/plat-imx/imx_caam.h

```diff
@@ -14,14 +14,23 @@
 struct imx_caam_job_ring {
 	uint32_t			jrmidr_ms;
 	uint32_t			jrmidr_ls;
-};
+} __packed;
```

Comment:
Should be part of "plat-imx: pack caam structs" I suppose

Reply:
I will fix that while squashing the review commits.